app/views/analytics/_customer_analytics.haml

```diff
@@ -0,0 +1,38 @@
+- if feature_enabled?(:customer_universal_analytics)
+  :javascript
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','//www.google-analytics.com/analytics.js','ga_customer');
+
+    var community_ga_key = "#{@current_community.google_analytics_key}"
+    var community_cookie_domain = "#{PublicSuffix.parse(request.host).domain}"
+    ga_customer('create', community_ga_key, 'auto', {'legacyCookieDomain': community_cookie_domain,   'allowLinker': true});
+    ga_customer('send', 'pageview');
+    var secondary_analytics_in_use = true;
+
+    var customer_report_event = function(category, action, opt_label) {
+      if (typeof ga_customer === 'function'){
+        ga_customer('send', 'event', category, action, opt_label);
+      }
+    }
+- else
+  :javascript
```

Comment:
I'd suggest that this would also be splitted to 2 partials: _customer_legacy_analytics & _customer_universal_analytics
It would make it more clear what to remove when legacy is no longer needed.